bayesian_cnn_prometheus/constants.py

```diff
@@ -17,9 +17,9 @@ class DatasetTypeTargets:
 
 class Paths:
     PROJECT_DIR = Path(bayesian_cnn_prometheus.__file__).parent
-    DATA_DIR = PROJECT_DIR / "data"
+    DATA_DIR = PROJECT_DIR/'data'
 
-    CONFIG_PATH = PROJECT_DIR / 'config.json'
+    CONFIG_PATH = PROJECT_DIR / f'config.local.json'
```

Comment:
Dlaczego f string? Dlaczego z kropką? Dla mnie to wygląda jak inne rozszerzenie :D Chyba że tak się powinno robić. :)

Reply:
> Dlaczego f string
Głównie dlatego, że jak coś zaczyna się na `f` to z definicji powinno być dobre ;) 

> Dlaczego z kropką?
Taki pattern spotkałem w C#. Według mnie jest całkiem wygodny, ale równie dobrze możemy nazwać w kebab-casie. Albo po prostu `config`, jak to zwykle robili slawni developerzy utylit dla Linuxa ;)